django/bossspatialdb/views.py

```diff
@@ -234,7 +237,7 @@ class Downsample(APIView):
     * Requires authentication.
     """
     # Set Parser and Renderer
-    parser_classes = (JSONRenderer, BrowsableAPIRenderer)
+    parser_classes = (JSONParser, BrowsableAPIRenderer)
```

Comment:
I am just trying to understand the code a little better.  From my reading JSONParser takes JSON to Dictionary. Why wasn't this always a JSONParser instead of a JSONRenderer?